src/app/docs/streaming-chat/page.md

```diff
@@ -168,6 +168,8 @@ Add the Chat controller:
 ```ruby
 # app/controllers/chats_controller.rb
 class ChatsController < ApplicationController
+  respond_to :html
```

Comment:
Otherwise it fails with `In order to use respond_with, first you need to declare the formats your controller responds to in the class level.`.